xdsl/ir.py

```diff
@@ -515,27 +516,6 @@ def __hash__(self) -> int:
         ...
 
 
-@dataclass(frozen=True)
-class OpTrait:
```

Comment:
Why did you decide to move this?

Reply:
There was a circular dependency on the import between `ir.py` and `traits.py`.